evap/staff/templates/staff_base.html

```diff
@@ -2,11 +2,17 @@
 
 {% block title %}{% trans 'Manage' %} - {{ block.super }}{% endblock %}
 
-{% block breadcrumb %}
-    {{ block.super }}
-    {% if disable_breadcrumb_staff %}
-        <li class="breadcrumb-item">{% trans 'Manage' %}</li>
-    {% else %}
-        <li class="breadcrumb-item"><a href="{% url 'staff:index' %}">{% trans 'Manage' %}</a></li>
-    {% endif %}
+{% block breadcrumb_bar %}
+    <div>
+        <ul class="breadcrumb">
+            {% block breadcrumb %}
+                {% if disable_breadcrumb_staff or not user.is_staff %}
+                    <li class="breadcrumb-item">{% trans 'Manage' %}</li>
```

Comment:
same here, when is this shown to non-staff users?

Reply:
for reviewers :)